antiplag/tasks.py

```diff
@@ -34,28 +45,30 @@ def compare_documents(documents):
             try:
                 # returns percentage representing how similar docs are
                 similarity = text_comparison(doc.text, similar_doc["text"])
+                result_similarity += similarity
             except:
-                # Ignore document
-                pass
+                similarity = None
+                result_similarity += 1
```

Comment:
Why are we raising the similarity by one if text comparison returned error?

Reply:
Fixed, similarity should not be raised.